apps/dev-recruiters/src/components/modules/TalcommunityPage/TalFormPage/signUpForm.tsx

```diff
@@ -157,13 +158,6 @@ export default function TalCommForm({ handleCloseModal, position }: Props) {
                       onChange={handleSetCheckCheckbox}
                       required
                     />
-                    <Checkbox
-                      label="I have read and agreed to the terms and conditions"
```

Comment:
Why remove this checkbox? I think it's nice to have it.
Applying without agreeing to the terms and conditions might become problematic down the line.

Reply:
After the review, I added the Terms and Conditions checkbox and reverted the commented lines on the 'Second' page to their previous state.